server/models/client_info_result.js

```diff
@@ -11,6 +11,14 @@ class ClientInfoResult {
     this.address = address;
     this.phoneNumber = phoneNumber;
     this.wallet = wallet;
+    this.hasPendingCancelation = false;
+  }
```

Comment:
It should also have: hasNotCovered

Reply:
It should be done by the one who is implementing the insufficient wallet BE task